### PR TITLE
fix(platform): fix a11y issue for platform object attribute component

### PR DESCRIPTION
**Pull request by manu-kr**
## Related Issue.
Closes SAP/fundamental-ngx/https://github.com/SAP/fundamental-ngx/issues/5767

## Description

This PR change contains to fixes a11y issues for the platform object attribute component.

##### During Implementation
1. Visual Testing:
- [x] visual misalignments/updates
- [x] check Light/Dark/HCB/HCW themes
- [x] RTL/LTR - proper rendering and labeling
- [x] responsiveness(resize)
- [x] Content Density (Cozy/Compact/(Condensed))
- [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [x] Mouse vs. Keyboard support
- [x] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality
- [x] the commit message(s) follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review
- [x] visual misalignments/updates
- [x] check Light/Dark/HCB/HCW themes
- [x] RTL/LTR - proper rendering and labeling
- [x] responsiveness(resize)
- [x] Content Density (Cozy/Compact/(Condensed))
- [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [x] Mouse vs. Keyboard support
- [x] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

File: libs/platform/src/lib/object-attribute/object-attribute.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {{label}}
 <ng-content></ng-content>
 <ng-container *ngIf="linkText !== null && linkText !== undefined">
-    <span (click)="onObjectAttributeClick($event)" tabindex="0" [attr.aria-label]="linkText" [attr.title]="linkText"
-        [ngClass]="islink? 'fd-object-attribute--link': ''" role="link" [attr.aria-disabled]="disabled"
-        [attr.disabled]="disabled">
+    <span (click)="onObjectAttributeClick($event)" [attr.tabindex] = " !disabled ? '0' : '-1' " [attr.aria-label]="linkText" [attr.title]="linkText"
```
Comment (N1XUS) [inline libs/platform/src/lib/object-attribute/object-attribute.component.html:4]: Please remove spaces:
```
[attr.tabindex]="disabled ? '-1' : '0'"
```

Comment (manu-kr) [reply libs/platform/src/lib/object-attribute/object-attribute.component.html]: Modified

**Review by mikerodonnell89 — APPROVED** [review]
LGTM

**Review by Lokanathan-k — APPROVED** [review]
LGTM

**Comment by puru-hk** [comment]
all issues fixed.

**Review by puru-hk — APPROVED** [review]

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

Issue 1: Fixed.

Issue 2: Fixed.

Issue 3:  Fixed.

Issue 4: Fixed.

issue 5: Fixed.

Issue 6: Fixed.

Retest Details:
https://deploy-preview-6473--fundamental-ngx.netlify.app/#/platform/object-attribute
JAWS 2021

**Pull request merged**